public/locales/en/files.json

```diff
@@ -0,0 +1,62 @@
+{
+  "title": "Files",
+  "home": "Home",
+  "breadcrumbs": "Breadcrumbs",
+  "finished": "Finished!",
+  "totalSize": "Total size: {size}",
+  "filesSelected": "{count, plural, one {File selected} other {Files selected}}",
```

Comment:
I also changed this one, because it supported only 2 plural forms, now it is universal :)